### PR TITLE
Microwaved snacks

### PR DESCRIPTION
**Pull request by Meatstuff882**
# About the pull request

Lets you microwave snacks. Updates the inspect text on these snacks to suggest microwaving them. Microwaving snacks adds 2 points of nutrition. All snacks (including donk pockets) now provide the same amount of nutrition.

# Explain why it's good for the game

The food said that it didn't have a time to microwave them for. Marines should be allowed to microwave them anyways without ruining the microwave. Also adds cheese to packaged cheeseburgers because they didn't have it before.

# Testing Photographs and Procedure
image
This is what losing looks like.

</details>

# Changelog

:cl:
add: Added Microwaved Burritos, Cheeseburgers, and Hot Dogs, and recipes for these items.
balance: Packaged food has more nutrition.
:cl:

### CODE REVIEW

**Review by Doubleumc — DISMISSED** [review]

File: code/datums/recipe.dm [hunk 1]
```diff
@@ -654,6 +654,81 @@
 
 //other
 
+/datum/recipe/packaged_burrito/proc/warm_up(obj/item/reagent_container/food/snacks/packaged_burrito/being_cooked)
```
Comment (AndroBetel) [inline code/datums/recipe.dm:657]: copypasting this proc for 3 food types is really lazy, what you should do is create a subtype (/datum/recipe/**warmable**/burrito for example?) and make it have all the necessary procs

File: code/datums/recipe.dm [hunk 2]
```diff
@@ -654,6 +654,81 @@
...
+	being_cooked.reagents.add_reagent("meatprotein", 1)
+	being_cooked.name = "Warm " + being_cooked.name
+	being_cooked.desc = "A hard microwaved burrito. Molten on the outside, barely cooked on the inside."
+
+/datum/recipe/packaged_burrito/make_food(obj/container as obj)
+	var/obj/item/reagent_container/food/snacks/packaged_burrito/being_cooked = ..(container)
+	warm_up(being_cooked)
+	return being_cooked
+
+/datum/recipe/packaged_burrito/warm
```
Comment (AndroBetel) [inline code/datums/recipe.dm:669]: why is it a subtype, exactly? parent type doesnt have any requirements for cooking too

**Review by AndroBetel — CHANGES_REQUESTED** [review]

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Review by AndroBetel — DISMISSED** [review]

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Pull request merged**